src/ApiResourceExampleDocument.js

```diff
@@ -313,6 +317,25 @@ export class ApiResourceExampleDocument extends AmfHelperMixin(LitElement) {
     }));
   }
 
+  get _collapseExamplePanel() {
+    return this.__collapseExamplePanel
+  }
+
+  set _collapseExamplePanel(value) {
+    const old = this.__collapseExamplePanel;
+    if (old === value) {
+      return;
+    }
+    this.__collapseExamplePanel = value;
+    this.requestUpdate('_collapseExamplePanel', old);
+    this.dispatchEvent(new CustomEvent('collapse-example-panel-changed', {
```

Comment:
What would this new event be for?

Reply:
Would be when shot the click  event by user in the example-title area, 

Is my approach correct?

Reply:
Talking with Caro, She explain me that this method is just necessary when the event is shot from external component, in this case no it es necessary, so that I remove it